src/KronFluxAlgorithm.cc

```diff
@@ -476,29 +412,56 @@ getPsfFactor(CONST_PTR(afw::detection::Psf) psf,
                                           afw::geom::Point2D(psfXCen, psfYCen));
     return photometer(*mi, aperture, maxSincRadius).first;
 }
+} // anonymous namespace
 
 /************************************************************************************************************/
 
-template <typename PixelT>
-void KronFlux::_applyAperture(
+/**
+ * @brief A class that knows how to calculate fluxes using the KRON photometry algorithm
+ *
+ * @ingroup meas/algorithms
+ */
+KronFluxAlgorithm::KronFluxAlgorithm(KronFluxControl const & ctrl, std::string const & name, afw::table::Schema & schema) :
+    _name(name),
+    _ctrl(ctrl),
+    _fluxResultKey(
+        meas::base::FluxResultKey::addFields(schema, name, "flux from Kron Flux algorithm")
+    ),
+    _radiusKey(schema.addField<float>(name + "_radius", "Kron radius (sqrt(a*b))")),
+    _radiusForRadiusKey(schema.addField<float>(name + "_radiusForRadius",
+                                      "Radius used to estimate <radius> (sqrt(a*b))")),
+    _centroidExtractor(schema, name, true)
+{
+    static boost::array<meas::base::FlagDefinition,N_FLAGS> const flagDefs = {{
+        {"flag", "general failure flag, set if anything went wrong"},
```

Comment:
I actually think that Jim's example in PsfFlux.cc has lower case.  The upper case was from the older meas_algorithm doc strings, so I am taking lower case to be the rule and changing others to match "general".